code/core/src/phone/java/com/adobe/marketing/mobile/MobileCore.java

```diff
@@ -939,6 +940,29 @@ public static void trackState(
         dispatchEvent(event);
     }
 
+    // ========================================================
+    // Messaging Delegate methods
+    // ========================================================
+
+    /**
+     * Gets a previously set Message delegate.
+     *
+     * @return {@link MessagingDelegate} used to listen for current message lifecycle events
+     */
+    @Nullable public static MessagingDelegate getMessagingDelegate() {
```

Comment:
No big deal- but the spotless annotation does weird things when using NonNull annotation on methods. We might want to remove annotation processing it in the next versions.